main/tasks.py

```diff
@@ -273,3 +277,102 @@ def email_monthly_charges_report() -> None:
     email_monthly_charges_report_logic(
         last_month_start.month, last_month_start.year, last_month_name
     )
+
+
+def sync_clockify_time_entries() -> None:
+    """Task to sync time entries from Clockify API to TimeEntry model."""
+    api_key = os.getenv("CLOCKIFY_API_KEY")
+    if not api_key:
+        print("Clockify API key not found in environment variables")
+        return
+
+    days_back = 30
+    api = ClockifyAPI(api_key)
```

Comment:
Getting the Clockify API key should be done in the django settings, and here just use it, if available. Also, use the logger, not `print`:

Somewhere above:
```python
import logging
from django.conf import settings

logger = logging.getLogger(__name__)
```

And then:
```suggestion
    if not settings.CLOCKIFY_API_KEY:
        logger.warning("Clockify API key not found in environment variables")
        return

    days_back = 30
    api = ClockifyAPI(settings.CLOCKIFY_API_KEY)
```